src/TheThingsNetwork.cpp

```diff
@@ -357,15 +357,21 @@ void TheThingsNetwork::clearReadBuffer()
   }
 }
 
-size_t TheThingsNetwork::readLine(char *buffer, size_t size)
+size_t TheThingsNetwork::readLine(char *buffer, size_t size, uint8_t attempts = 3) // Default timeout value 10s and is set in class initiator
```

Comment:
Where is it set?

Reply:
timeout is set in:
https://github.com/TheThingsNetwork/arduino-device-lib/blob/4629038364b748452bad617d008442f15e948238/src/TheThingsNetwork.cpp#L294

Reply:
Set defaults in the header file